platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/attributes/ClientAttributes.kt

```diff
@@ -61,6 +63,9 @@ internal class ClientAttributes(
             // A positive integer used as an internal version number.
             // This number helps determine whether one version is more recent than another.
             "_app_version_code" to appVersionCode.toString(),
+            // Installation source of the current app (for example if the app is installed from play
+            // store this value will return `com.android.vending`)
+            "_app_installation_source" to installationSource,
```

Comment:
is it necessary to report it on every single event if the value doesn't change? wouldn't it be better to just have it as part of the SDKConfigured log?